frontend/src/pages/OnboardingFlow/OnboardingModal/Steps/DataHandling/index.jsx

```diff
@@ -173,48 +173,51 @@ function DataHandling({ nextStep, prevStep, currentStep }) {
             ))}
           </div>
         </div>
-
-        <div className="flex flex-col gap-y-3.5 border-b border-zinc-500/50 pb-8">
+        <div className="flex flex-col gap-y-2 border-b border-zinc-500/50 pb-4">
           <div className="text-white text-base font-bold">Embedding Engine</div>
           <div className="flex items-center gap-2.5">
             <img
               src={EMBEDDING_ENGINE_PRIVACY[embeddingEngine].logo}
-              alt="Vector DB Logo"
+              alt="LLM Logo"
               className="w-8 h-8 rounded"
             />
             <p className="text-white text-sm font-bold">
               {EMBEDDING_ENGINE_PRIVACY[embeddingEngine].name}
             </p>
           </div>
-          <ul className="flex flex-col list-disc">
+          <div className="flex flex-col">
             {EMBEDDING_ENGINE_PRIVACY[embeddingEngine].description.map(
               (desc) => (
-                <li className="text-white/90 text-sm">{desc}</li>
+                <p className="text-white/90 text-sm">
+                  <b>•</b> {desc}
+                </p>
               )
             )}
-          </ul>
+          </div>
```

Comment:
Why cant we just keep using a `<ul>` here with `list-disc` which will add the dots for us?